SEC/Equities/EquityInstruments.rdf

```diff
@@ -530,10 +658,29 @@
 				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-aeq;ShareholdersEquity"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;isClassifiedBy"/>
```

Comment:
again I'm not keen on use of isClassifiedBy

Reply:
fixed - see above